### PR TITLE
rebranding and internationalization

### PR DESCRIPTION
**Pull request by rmarinleal**
* Renamed module to Pagantis
* Translated to IT and FT
* Internationalization

### CODE REVIEW

File: pagantis/pagantis.php [hunk 1]
```diff
@@ -96,14 +102,14 @@ function plgVmConfirmedOrder ($cart, $order) {
         $jinput = JFactory::getApplication()->input;
 
         //Account Settings
-        $environment = $method->pagamastarde_env;
-        $discount = $method->pagamastarde_discount;
+        $environment = $method->pagantis_env;
+        $discount = $method->pagantis_discount;
         if($environment == 'test'){
```
Comment (pgarcess) [inline pagantis/pagantis.php]: Improve the admin panel to show only a field for the public_key and other one for secret_key, like we are doing in the rest of the modules

File: pagantis/update.xml [hunk 2]
```diff
@@ -9,17 +25,17 @@
...
          <tag>stable</tag>
      </tags>
      <maintainer>Albert Fatsini</maintainer>
-     <maintainerurl>https://www.pagamastarde.com/</maintainerurl>
+     <maintainerurl>https://www.pagantis.com/</maintainerurl>
      <targetplatform name="joomla" version="2.5"/>
   </update>
   <update>
-     <name>Pagamastarde</name>
+   <name>Paga+Tarde</name>
```
Comment (pgarcess) [inline pagantis/update.xml:38]: Pagantis

File: pagantis/update.xml [hunk 3]
```diff
@@ -28,17 +44,17 @@
...
```
Comment (pgarcess) [inline pagantis/update.xml:57]: Pagantis

File: pagantis/update.xml [hunk 4]
```diff
@@ -47,17 +63,17 @@
...
```
Comment (pgarcess) [inline pagantis/update.xml:76]: Pagantis

File: pagantis/update.xml [hunk 5]
```diff
@@ -66,17 +82,17 @@
...
```
Comment (pgarcess) [inline pagantis/update.xml:95]: Pagantis

File: pagantis/update.xml [hunk 6]
```diff
@@ -85,17 +101,17 @@
...
```
Comment (pgarcess) [inline pagantis/update.xml:114]: PAgantis => Search in the whole code

**Review by pgarcess — CHANGES_REQUESTED** [review]

File: pagantis/pagantis.php (hunk 1 above)
Comment (rmarinleal) [reply pagantis/pagantis.php]: Done

File: pagantis/update.xml (hunk 6 above)
Comment (rmarinleal) [reply pagantis/update.xml:114]: No, this Paga+Tarde texts are for the Fatsini Pagamastarde releases. This file is a roadmap, and points to the pagamastarde versions, we can delete the history in we want, but don't have sense to replace the words

File: pagantis/update.xml (hunk 2 above)
Comment (rmarinleal) [reply pagantis/update.xml:38]: answered in the las coment.

File: pagantis/update.xml (hunk 3 above)
Comment (rmarinleal) [reply pagantis/update.xml:57]: answered in the las coment.

File: pagantis/update.xml (hunk 4 above)
Comment (rmarinleal) [reply pagantis/update.xml:76]: answered in the las coment.

File: pagantis/update.xml (hunk 5 above)
Comment (rmarinleal) [reply pagantis/update.xml:95]: answered in the las coment.

**Review by pgarcess — APPROVED** [review]

**Pull request merged**